internal/services/media/media_streaming_policy_resource.go

```diff
@@ -117,13 +255,32 @@ func resourceMediaStreamingPolicy() *pluginsdk.Resource {
 				MaxItems: 1,
 				Elem: &pluginsdk.Resource{
 					Schema: map[string]*pluginsdk.Schema{
+						"clear_key_encryption_configuration": {
```

Comment:
Configuration is implied, we should remove that from the name
```suggestion
						"clear_key_encryption": {
```